SW/main/main.py

```diff
@@ -0,0 +1,26 @@
+import firebase_admin
+import time
+from firebase_admin import credentials, db
+from fisheye_usage_3 import capture
+
+from Stove import Stove
+from Thermal import Thermal
+
+cred = credentials.Certificate('./ServiceAccountKey.json')
+default_app = firebase_admin.initialize_app(cred, {'databaseURL': 'https://the-rat-magic.firebaseio.com/'})
+
+db = db.reference("Stove")
+
+thermal = Thermal()
+stove = Stove()
+
+while (True):
+
+    capture()
```

Comment:
what is this doing? we do need to capture a picture so we can pass it along, but this isn't doing anything. won't this call imshow every single frame?

Reply:
I can take out imshow from the fisheye_usage code, but I think it would take too much time out of my other tasks to try and get undistort to take in an object instead. I can create a task for it later, but I don't think it's the best use of my time right now.

Reply:
good call. to resolve...

- turn path into a variable
- return path from capture function call
- add load call to main file